WARNING | 2021-12-01 22:55:09,390 | cle.backends.externs | Symbol was allocated without a known size; emulation may fail if it is used non-opaquely: gnutls_free
WARNING | 2021-12-01 22:55:09,399 | cle.loader | For more information about "Symbol was allocated without a known size", see https://docs.angr.io/extending-angr/environment#simdata
Fatal Python error: Cannot recover from stack overflow.

Current thread 0x00007f2c9d1d70c0 (most recent call first):
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/ast/base.py", line 307 in _arg_serialize
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/ast/base.py", line 316 in _arg_serialize
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/ast/base.py", line 316 in _arg_serialize
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/ast/base.py", line 316 in _arg_serialize
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/ast/base.py", line 335 in _ast_serialize
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/ast/base.py", line 266 in _calc_hash
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/ast/base.py", line 224 in __new__
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 515 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  ...
timeout: the monitored command dumped core
Aborted
